tlatools/org.lamport.tlatools/test/tla2sany/semantic/TestLevelChecking.java

```diff
@@ -67,7 +78,7 @@ public String summarize(Errors log) {
      * operator synonym resolution should happen before any operator details
      * are fetched. Here we test whether that hypothesis is true.
      */
-    new LevelCheckingTestCase("x == \\neg TRUE", true),
+    new LevelCheckingTestCase("\\neg c",         true),
```

Comment:
Why does the expression change? Additionally, there appears to be unnecessary indentation.

Reply:
I just intended so all the booleans are lined up in the array. The expression changed because the string I was using for interpolation changed, adding constant & variable definitions as well as supplying an operator body to fill in.